test/cpp/util/test_credentials_provider.cc

```diff
@@ -19,19 +19,23 @@
 
 #include "test/cpp/util/test_credentials_provider.h"
 
+#include <gflags/gflags.h>
+#include <grpc/support/log.h>
+#include <grpc/support/sync.h>
+#include <grpcpp/security/server_credentials.h>
+
 #include <cstdio>
 #include <fstream>
 #include <iostream>
-
 #include <mutex>
 #include <unordered_map>
 
-#include <gflags/gflags.h>
-#include <grpc/support/log.h>
-#include <grpc/support/sync.h>
-#include <grpcpp/security/server_credentials.h>
-
-#include "test/core/end2end/data/ssl_test_data.h"
+#include "src/core/lib/iomgr/load_file.h"
```

Comment:
It seems that you did not run tools/dockerfile/grpc_clang_format/clang_format_all_the_things.sh to format your change. 
Run
`CLANG_FORMAT_ROOT=~/grpc tools/dockerfile/grpc_clang_format/clang_format_all_the_things.sh`